concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetPoolDelegators.java

```diff
@@ -0,0 +1,52 @@
+package com.concordium.sdk.examples;
+
+import com.concordium.sdk.ClientV2;
+import com.concordium.sdk.Connection;
+import com.concordium.sdk.Credentials;
+import com.concordium.sdk.exceptions.ClientInitializationException;
+import com.concordium.sdk.requests.BlockHashInput;
+import com.concordium.sdk.responses.AccountIndex;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.concurrent.Callable;
+
+@Command(name = "GetPoolDelegators", mixinStandardHelpOptions = true)
+public class GetPoolDelegators implements Callable<Integer> {
+    @Option(
+            names = {"--endpoint"},
+            description = "GRPC interface of the node.",
+            defaultValue = "http://localhost:20002")
+    private String endpoint;
+
+    @Option(
+            names = {"--bakerId"},
+            description = "Baker Id",
+            defaultValue = "2924")
```

Comment:
There should not be a default for this value. There is no sensible default.

Reply:
done. The idea of a default value here was to enable the developers checking out this repo to just use their IDE to execute an example.